src/Test/Plutip/Internal/LocalCluster.hs

```diff
@@ -234,11 +195,18 @@ data ClusterStatus (a :: Type)
  cluster alive until the ClusterClosing action is called.
 -}
 startCluster :: forall (a :: Type). ReaderT ClusterEnv IO a -> IO (TVar (ClusterStatus a), a)
```

Comment:
I think it would be cleaner, if we just required the `PlutipConfig` everywhere, maybe except for `withCluster`